ui/components/src/InfoTooltip/InfoTooltip.test.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { act, render } from '@testing-library/react';
+import { fireEvent, screen } from '@testing-library/dom';
+import { InfoTooltip } from './InfoTooltip';
+
+describe('InfoTooltip', () => {
+  const title = 'Tooltip Title';
+  const description = 'this is a tooltip';
+  it('should render title and description', async () => {
+    render(
+      <InfoTooltip title={title} description={description}>
+        <div>tooltipAnchor</div>
+      </InfoTooltip>
+    );
+    const tooltipAnchor = screen.getByText('tooltipAnchor');
+    act(() => {
+      fireEvent(
+        tooltipAnchor,
+        new MouseEvent('mouseover', {
+          bubbles: true,
+        })
+      );
+    });
+    await screen.findByText(title);
+    await screen.findByText(description);
```

Comment:
Nit: does this need to be async also since you already awaited for the title to show? (Just a minor thing that might keep tests faster if it doesn't need to be the async version).